lib/plugins/http-error-handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function httpErrorHandler(request, reply) {
+  const response = request.response;
+  if (response.isBoom) {
+    let msg;
+    let statusCode = response.output.payload.statusCode;
+    let errorTitle = response.output.payload.error;
+    let errorMessage = response.output.payload.message;
+
+    switch(statusCode) {
+      case 404:
+        errorMessage = 'Sorry, that page is not available.';
```

Comment:
I liked the idea implemented in hapi-error of producing friendlier messages for some errors, rather than having a dull "Not Found" in there. Which is only applicable when using them in templates, as they are perfectly fine for standard API usage. Does it make sense?